articles/introduction.html

```diff
@@ -96,21 +96,22 @@
 </p>
 
 <p>
-Currently, the most popular Go compiler is maintained by Go design team.
-Later we will call it the standard Go compiler, or <code>gc</code>
-(an abbreviation for Go compiler, not for garbage collection).
-Go design team also maintains another Go compiler, <code>gccgo</code>,
-which is used less popular than <code>gc</code>.
-The two compilers are still in active development.
-But Go team focuses more on the development of <code>gc</code>.
+Currently, the most popular Go compiler is written in Go and maintained by the Go design team.
+Later we shall call it the standard Go compiler, or <code>gc</code>
+(an abbreviation for Go compiler, not for garbage collection GC).
+The Go design team also maintains another Go compiler, <code>gccgo</code>,
+which was initially used to bootstrap the language.
+Nowadays it's use is is becoming less popular than <code>gc</code>.
+Both compilers are in active development.
+As of now the Go team focuses on the improvement of <code>gc</code>.

```

Comment:
gccgo is not the compiler to bootstrap Go.
The early gc versions (1.4-) are written with C, which has no relations to gccgo.
Gccgo is mainly used as a sanity check for gc, making sure they both implement the documented specification, rather than a single implementation becoming the specification.

Reply:
Hm - I remembered differently, and I remembered wrong. 

Yes, Ian Lance Taylor added go to the GNU c++ compiler gcc in 2009/2010 and published 

https://talks.golang.org/2010/gofrontend-gcc-summit-2010.pdf

The history is also here

https://talks.golang.org/2015/keeping-up.slide
(which is @ https://github.com/golang/talks/blob/master/2015/keeping-up.slide)

I wonder if You like to incorporate links such as these into Your book.

Reply:
No problem, in fact, I plan to add more external links in other articles.